example/android/build.gradle

```diff
@@ -0,0 +1,17 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.1.2'
+        classpath 'org.jsonschema2dataclass:jsonschema2dataclass-gradle-plugin:latest.integration'
```

Comment:
What is right here ?
@eirnym

Reply:
should be `gradle.plugin.org.jsonschema2dataclass:jsonschema2dataclass`